server/stage.go

```diff
@@ -47,10 +47,14 @@ func (s *Server) composeStageRelationship(ctx context.Context, stage *api.Stage)
 			return err
 		}
 	} else {
-		for _, task := range stage.TaskList {
-			if err := s.composeTaskRelationship(ctx, task); err != nil {
+		for i, task := range stage.TaskList {
+			// TODO(dragonly): remove this hack
```

Comment:
pls elaborate it.

Reply:
As is discussed with @d-bytebase, I'll propose another PR to decouple validate only check and the compose logic of `Issue`. That "another PR" will block this PR. Then this line will hopefully not exist.

Reply:
Fixed.